lib/membership/member.js

```diff
@@ -79,40 +79,10 @@ Member.prototype.decayDampScore = function decayDampScore() {
     this.emit('dampScoreDecayed', this.dampScore, oldDampScore);
 };
 
-// This function is named with the word "evaluate" because it is not
-// guaranteed that the update will be applied. Naming it "update()"
-// would have been misleading.
-Member.prototype.evaluateUpdate = function evaluateUpdate(update) {
-    // The local override and "other" override rules that are evaluated
-    // here stem from the rules defined in the SWIM paper. They deviate
-    // a bit from that literature since Ringpop has added the "leave"
-    // status and retains faulty members in its membership list.
-    if (this._isLocalOverride(update)) {
-        // Override intended update. Assert aliveness!
-        this.ringpop.stat('increment', 'refuted-update');
-        var newIncNumber = this.Date.now();
-        update = {
-            source: this.ringpop.whoami(),
-            sourceIncarnationNumber: newIncNumber,
-            address: this.address,
-            status: Member.Status.alive,
-            incarnationNumber: newIncNumber
-        };
-    } else if (!this._isOtherOverride(update)) {
-        return;
-    }
-
-    // We've got an update. Apply all-the-things.
-    var oldStatus = this.status;
+// We've got an update. Apply all-the-things.
+Member.prototype.applyUpdate = function applyUpdate(update) {
     if (this.status !== update.status) {
```

Comment:
I think we could remove the if statement for readability as well. Both with and without this check the value of `this.status` will be equal to `update.status`.